NewForce-Cohort-9/newforce-c9-tabloid-cli-TabloidCLI
# Additions & logic for Blogs, Blog Menu, started Blog Details

# Description

This implements CLI logic to accommodate for Blogs. It creates BlogRepository, BlogManager, and completes logic for allowing users to view, add, edit, and delete blog entries.  It also adds BlogDetailManager so that a nested menu appears when Blog Details is chosen, although the logic isn't fully implemented for this yet. 

## Type of change
New feature (non-breaking change which adds functionality)

# Testing Instructions

- Start TabloidCLI.sln
- Run program
-  From console menu, enter 2 to enter Blog Menu

 From the Blog Menu: 
- User should be able to press 1 to view a list of blogs.
- User should be able to press 2 to enter Blog Details menu (menu should be viewable only)
- User should be able to press 3 to add a new blog. 
- User should be able to press 4 to edit an existing blog.
- User should be able to press 5 to remove an existing blog.
- User should be able to press 0 to return to the main menu.

Note that BlogDetailManager should appear when entering 2, although the functionality for the BlogDetailManager is in progress and not fully implemented. 


# Checklist:
